schemas/iso19139/src/main/plugin/iso19139/update-fixed-info.xsl

```diff
@@ -770,4 +770,13 @@
     </xsl:if>
   </xsl:template>
 
+  <!-- gmd:dateTime requires gco:DateTime -->
+  <xsl:template match="gmd:dateTime" priority="200">
+    <xsl:variable name="value" select="gco:Date|gco:DateTime" />
```

Comment:
why are you checking for gco:Date here, gmd:dateTime should not contain a gco:Date?

Reply:
As indicated in the description of the pull request:

> Date fields in the metadata editor use the directive gn-date-picker, this directive adds for ISO19139 the element gco:Date if the time part is set to empy, but gmd:dateTime requires gco:DateTime (see http://www.datypic.com/sc/niem21/e-gmd_dateTime-1.html).